src/Microsoft.Fx.Portability/DataExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static class DataExtensions
 
         public static JsonSerializerSettings JsonSettings { get; } = new JsonSerializerSettings
         {
-            Formatting = Formatting.Indented,
+            Formatting = Formatting.None,
```

Comment:
The indented is nice for the times when we want to look at it. I'm fine with Formatting.None, but I'm wondering what the size difference is with this especially when the stream gets compressed.